code/string-calculator/src/main/java/ru/unn/agile/calc/model/StringCalc.java

```diff
@@ -0,0 +1,96 @@
+package ru.unn.agile.calc.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class StringCalc {
+    public StringCalc() {
+
+    }
+
+    public double result(final String string) {
+        return parseString(string);
+    }
+
+    private double parseString(final String string) {
+        try {
+            char firstSymbol = string.charAt(0);
+            String[] parsedNumbers = string.split("\\+|\\-|\\*|\\/");
+            String[] parsedOperators = string.split("[1234567890\\.a-z]*");
+            int index = 0;
+            if (firstSymbol == '-') {
+                index++;
+                parsedNumbers[index] = "-" + parsedNumbers[index];
+                parsedOperators[0] = "";
+            }
+
+            List<Integer> priorityList = new ArrayList<Integer>();
+            List<String> operationsList = new ArrayList<String>();
+
+            for (String operator : parsedOperators) {
+                if (operator.equals("+")) {
```

Comment:
What about using constants for operators?

Reply:
Done